sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -47,6 +47,8 @@ export enum AppRoutes {
   InstitutionManageUsers = "manage-users",
   ActiveApplicationScholasticStandingView = "active-application/location/:locationId/scholastic-standing/:scholasticStandingId",
   OfferingsUpload = "offerings-upload",
+  InstitutionStudentDetail = "student-detail/:studentId",
+  InstitutionStudentProfile = "student-profile",
```

Comment:
IMO `profile`

Reply:
https://github.com/bcgov/SIMS/pull/1863#discussion_r1163338494

Reply:
Done for both institution and ministry.